src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -106,15 +75,21 @@ class TableView extends React.Component<TableViewProps, TableState> {
     const nextColumnSortBy = [...columnSortBy];
     nextColumnOrder[i] = nextColumn;
 
-    setColumnStateOnLocation(location, nextColumnOrder, nextColumnSortBy);
+    history.pushState({}, 'omg', '?woah=true');
+
+    if ((window as any).FOOOOOO) {
+      setColumnStateOnLocation(location, nextColumnOrder, nextColumnSortBy);
+    }
```

Comment:
Same here.

Reply:
@markstory good catch. 👍  I'll remove this.